beesdoo_product/views/beesdoo_product.xml

```diff
@@ -110,13 +110,15 @@
                     options="{'no_open': True, 'no_create': True, 'no_create_edit': True }"
                 />
                 <field name="purchase_price" />
+                <field name="purchase_price_write_date" />
                 <field
                     name="uom_po_id"
                     readonly="1"
                     options="{'no_open': True, 'no_create': True, 'no_create_edit':True }"
                 />
                 <field name="suggested_price" readonly="1" />
                 <field name="list_price" readonly="1" />
+                <field name="list_price_write_date" />
```

Comment:
these new fields take a lot of space (width) in the list. shorter names would be useful. since they are next to their corresponding prices, maybe simply “Last Update” would be enough (set with the `string` property directly here in xml)?

also, i don’t think displaying seconds is useful (and makes it wider), but i don’t know whether this can be easily changed.

Reply:
Tried in 77c6919. I was unable to remove the seconds.

Old (before PR):

![Capture d’écran de 2022-03-17 10-08-34](https://user-images.githubusercontent.com/12065945/158775886-6e410684-2479-4c59-8aec-dbc5272e87dd.png)

New (after 77c6919):

![Capture d’écran de 2022-03-17 10-05-10](https://user-images.githubusercontent.com/12065945/158775844-6caac391-44cb-4907-b80b-8659cffc5bed.png)